tests/python/requirements.txt

```diff
@@ -1,7 +1,7 @@
 pytest==6.2.5
 pytest-cases==3.6.13
 pytest-timeout==2.1.0
-pytest-cov==4.0.0
+pytest-cov==4.1.0
```

Comment:
This comment isn't directly related to this PR, but I'm not quite sure why we need it here.

As I understand we calculate coverage inside our containers and our cvat server has coverage dependency for it.
After coverage is calculated and written we move generated coverage files from containers to the local filesystem and then send it to codecov.

What is the point of having pytest-cov and running tests with `--cov` argument?

Reply:
We get code coverage for sdk and cli with this.